app/controllers/groups_controller.rb

```diff
@@ -51,7 +51,7 @@ def leave
       end
       @group.destroy
     end
-    redirect_to root_path
+    redirect_to groups_path
```

Comment:
This redirect is fine for people who leave their parent community group but won't make sense for people who leave the mentor program. Please write a custom redirect for people leaving the mentor program or redirect to root_path for everyone